dashboard/test/dsl/script_dsl_test.rb

```diff
@@ -722,6 +722,70 @@ def setup
     assert_equal expected, output
   end
 
+  test 'Script DSL for lesson group without lesson' do
```

Comment:
I didn't change any behavior here but figured a unit test was worth adding as we allow this scenario now